### PR TITLE
Mauxite Construction Sprites

### PR DESCRIPTION
**Pull request by Flaborized**
[FEAT]

## About the PR 

This PR makes it so some stuff made out of mauxite uses unique sprites instead of the material color being applied to the base. This includes:

- Walls + Rwalls
- Floors (Though floors with non-default patterns will currently use the patterned sprite. Might need to be fixed idk?)
- Lockers
- Stools
- Material sheets, rods, floortiles (Currently doesn't work, needs some code fix that I dunno how to do)
- Chairs
- Tables

Also change the default mauxite color to match these sprites

## Why's this needed? 

I think this is a fun way to implement new building options, while also giving some aesthetic values to materials. Mauxite is just an arbitrary 'first' thing to work on, I plan to (and others can also) add stuff for other appropriate building materials, so you can have lots of options and uses for all the stuff mining can procure. Also the rusty industrial look could be useful for some prefabs which I've seen mentioned/been asked about on discord. 

Works with the new fancy $$ material tag stuff that Amy coded and pali helped me out with. Very strong and useful toolkit for making this kinda stuff with more or less no additional code.

## Changelog 

```changelog
(u)Flaborized
(+)Added unique sprites for many construct-able things made out of mauxite sheets, for that industrial warehouse look. Includes walls, floors, tables, and chairs. Go build stuff with it! Mind the rust.
```

### CODE REVIEW

**Comment by BatElite** [comment]
Interesting changelog

**Comment by M-Earthfire** [comment]
How will this interact with alloys containing mauxite?

The sprites are lovely!

**Comment by Flaborized** [comment]
> Interesting changelog :P

Whoops, fixed!

> How will this interact with alloys containing mauxite?
> 
> The sprites are lovely!

It won't change sprites of alloys, just pure mauxite. It *might* change them if you make an alloy that's named 'mauxite', but then it won't inherit the alloy color or anything. But why would one do that !

**Comment by ZeWaka** [comment]
if it's wip,

**Comment by WALPVRGIS** [comment]
Cool as hell sprites Flabo :000000 !!

**Comment by Flaborized** [comment]
Added tables, chairs, changed the material color to match the rest of everything. (So objects without unique sprites will be closer looking to these bespoke ones) Also the sheets and rods and tiles weren't broken actually! With all that I think the PR is done, I dunno how to un-draft a PR (or if I can) but I think these are the features I wanna put in fully

image

File: code/modules/materials/Mat_Materials.dm [hunk 1]
```diff
@@ -363,7 +363,7 @@ ABSTRACT_TYPE(/datum/material/metal)
 	mat_id = "mauxite"
 	name = "mauxite"
 	desc = "Mauxite is a sturdy common metal."
-	color = "#574846"
+	color = "#534747
```
[the remark this answers is not in the document]
Comment (Flaborized) [reply code/modules/materials/Mat_Materials.dm]: Whoops! Fixed, thanks for pointing that out :shelterfrog:

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**